Thanks @crobert-1.  I've made the changes to the `elasticsearchexporter`'s `metadata.yaml` file in 6f11e12c05.  However, when I then ran `GITHUB_TOKEN=github_pat_XXXXXXX make update-codeowners` (from the repo's root directory), I got the following error:

```
$  GITHUB_TOKEN=github_pat_XXXXXXX make update-codeowners
cd cmd/githubgen && go install .
githubgen
2024/04/04 10:46:06 codeowners are not members: BinaryFissionGames, Frapschen, JonathanWamsley, MovieStoreGuy, Mrod1598, RichieSams, StefanKurek, VihasMakwana, adrielp, ag-ramachandran, akats7, alexvanboxel, antonblock, armstrmi, atingchen, atoulme, avadhut123pisal, bertysentry, braydonk, bryan-aguilar, chrroberts-pure, codeboten, cparkins, cpheps, damemi, dashpole, dehaansa, dgoscn, eedorenko, erichsueh3, evan-bradley, fatsheep9146, fchikwekwe, gjanco, greatestusername, hickeyma, hughesjj, jacobmarble, jamesmoessis, jmacd, jsuereth, kentquirk, khyatigandhi6, kkujawa-sumo, kongluoxing, liustanley, mapno, mar4uk, marctc, martin-majlis-s1, matej-g, mcardy, mcube8, moh-osman3, mrsillydog, nslaughter, pavankrish123, pcwiese, pjanotti, portertech, povilasv, psx95, pxaws, qiansheng91, rapphil, rmfitzpatrick, rnishtala-sumo, rogercoll, samiura, schmikei, sh0rez, shabicheng, shalper2, sinkingpoint, smithclay, splunkericl, srprash, swiatekm-sumo, tamir-michaeli, tomaz-s1, varunraiko, wangzlei, ycombinator, yotamloe, zdaratom-s1
make: *** [gengithub] Error 1
```

--- a/exporter/elasticsearchexporter/README.md
+++ b/exporter/elasticsearchexporter/README.md
@@ -6,7 +6,7 @@
 | Stability     | [beta]: traces, logs   |
 | Distributions | [contrib] |
 | Issues        | [![Open issues](https://img.shields.io/github/issues-search/open-telemetry/opentelemetry-collector-contrib?query=is%3Aissue%20is%3Aopen%20label%3Aexporter%2Felasticsearch%20&label=open&color=orange&logo=opentelemetry)](https://github.com/open-telemetry/opentelemetry-collector-contrib/issues?q=is%3Aopen+is%3Aissue+label%3Aexporter%2Felasticsearch) [![Closed issues](https://img.shields.io/github/issues-search/open-telemetry/opentelemetry-collector-contrib?query=is%3Aissue%20is%3Aclosed%20label%3Aexporter%2Felasticsearch%20&label=closed&color=blue&logo=opentelemetry)](https://github.com/open-telemetry/opentelemetry-collector-contrib/issues?q=is%3Aclosed+is%3Aissue+label%3Aexporter%2Felasticsearch) |
-| [Code Owners](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/CONTRIBUTING.md#becoming-a-code-owner)    | [@JaredTan95](https://www.github.com/JaredTan95) |
+| [Code Owners](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/CONTRIBUTING.md#becoming-a-code-owner)    | [@JaredTan95](https://www.github.com/JaredTan95), [@ycombinator](https://www.github.com/ycombinator) |
 
 [beta]: https://github.com/open-telemetry/opentelemetry-collector#beta
 [contrib]: https://github.com/open-telemetry/opentelemetry-collector-releases/tree/main/distributions/otelcol-contrib

